tests/test_cms.py

```diff
@@ -121,6 +121,64 @@ def modify_task(self):
             f.write("\n".join(sample))
 
 
+class TestNoLFInTextInput(TestSumCMS):
+    """Input without newline at the end with in_format=text."""
+
+    def expecting_success(self):
+        return False
+
+    def modify_task(self):
+        def modification_fn(raw_config):
+            raw_config["tests"]["in_gen"] = "gen_no_lf"
+            raw_config["tests"]["checker"] = ""
+
+        modify_config(self.task_dir, modification_fn)
+
+
+class TestNoLFInBinaryInput(TestSumCMS):
+    """Input without newline at the end with in_format=binary."""
+
+    def expecting_success(self):
+        return True
```

Comment:
Default implementation for this is same, so no reason to override it, right?

Reply:
This is for consistency with other "change something and run pisek" tests that set it (eg. [here](https://github.com/kasiopea-org/pisek/blob/42098fac2f0c99969dd1550f0bd0939e5d03c483/tests/test_kasiopea.py#L64)).

Reply:
Looks like we are not very consistent already :sweat_smile: